src/code2verus/processing.py

```diff
@@ -159,9 +161,11 @@ async def main_async(benchmark: str = "wendy-sun/DafnyBench", split: str = "test
     # Limit concurrent API calls to prevent rate limiting
     semaphore = asyncio.Semaphore(max_concurrent)
 
+    is_yaml = file_pattern == "*.yaml"
```

Comment:
In case the user passes in `subdir/*yaml` or something like that
```suggestion
    is_yaml = file_pattern.endswith("yaml")
```